plugins/scaffolder/src/api.ts

```diff
@@ -100,7 +100,7 @@ export class ScaffolderClient implements ScaffolderApi {
 
   async getIntegrationsList(options: { allowedHosts: string[] }) {
     const integrations = ScmIntegrations.fromConfig(
-      this.configApi.getConfig('integrations'),
+      this.configApi.getConfig(''),
```

Comment:
```suggestion
      this.configApi,
```

could you write a small test that exercises this as well?

Reply:
Updated config statement and added one test 😄